### PR TITLE
Tools dApp - Module Explorer UX/UI revamp - Editor

### PR DESCRIPTION
**Pull request by sanderlooijenga**
### Modify this title

This PR contains the revamped righthand side (editor) of the new UX/UI of the Modules Explorer.

### Test scenarios

- description of the (manually) executed test scenarios

### Reminders (if applicable)

- [x] I ran `pnpm install` and `pnpm test` in the root of the monorepo
      (optionally with `--filter=...package...` to exclude non-affected
      projects)
- [x] I ran `pnpm changeset` in the root of the monorepo
- [x] Test coverage has not decreased
- [ ] Docs have been updated to reflect changes in PR (don't forget
      docs.kadena.io)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207445755216491

### CODE REVIEW

File: packages/apps/tools/src/components/Global/ModuleExplorer/index.tsx [hunk 1]
```diff
@@ -60,45 +64,56 @@ const ModuleExplorer = ({
...
           );
-          // onTabClose(module);
+        }}
+        onModuleTabClose={(modules) => {
+          setOpenedModules(
+            openedModules.filter((openedModule) => {
+              return !modules.find((module) => {
+                return (
+                  `${openedModule.name}-${openedModule.chainId}-${openedModule.networkId}` ===
+                  `${module.name}-${module.chainId}-${module.networkId}`
```
Comment (r-mulder) [inline packages/apps/tools/src/components/Global/ModuleExplorer/index.tsx:106]: Bit of a nit but I would make a function for resolving the name, a small error could slip it and if the name ever changes, just one place to manage it.

File: packages/apps/tools/src/components/Global/ModuleExplorer/tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+      </Box>
+      <Stack
+        alignItems="center"
+        paddingInlineStart={'md'}
+        paddingBlockStart={'md'}
+        backgroundColor="surface.default"
+        className={secondLevelTabContainerStyles}
+      >
+        <Text className={tabsLabelStyles} size="smallest" bold>
+          Module on chain:
```
Comment (r-mulder) [inline packages/apps/tools/src/components/Global/ModuleExplorer/tabs.tsx:182]: Do we need to translate this one?

File: packages/apps/tools/src/components/Global/ModuleExplorer/index.tsx (hunk 1 above)
Comment (sanderlooijenga) [reply packages/apps/tools/src/components/Global/ModuleExplorer/index.tsx]: I've refactored it to use one function, great tip, thanks!

File: packages/apps/tools/src/components/Global/ModuleExplorer/tabs.tsx (hunk 2 above)
Comment (sanderlooijenga) [reply packages/apps/tools/src/components/Global/ModuleExplorer/tabs.tsx]: Good catch, made it use a translation entry now

**Review by r-mulder — APPROVED** [review]

**Pull request merged**